django_usaid_clinicfinder/django_usaid_clinicfinder/settings.py

```diff
@@ -0,0 +1,133 @@
+"""
+Django settings for django_usaid_clinicfinder project.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.7/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.7/ref/settings/
+"""
+
+import os
+
+import dj_database_url
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(__file__))
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.7/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'REPLACEMEPLSKTNKSBAI'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+TEMPLATE_DEBUG = True
+
+ALLOWED_HOSTS = ['localhost', '127.0.0.1', 'mmc.qa.praekelt.com']
```

Comment:
I think it's better not to set ``SECRET_KEY``, ``DEBUG``, ``TEMPLATE_DEBUG`` or ``ALLOWED_HOSTS`` at all here and only set them in production (I know setting this up with us is a bit of schlep).